clearpath_generator_robot/clearpath_generator_robot/launch/generator.py

```diff
@@ -142,6 +143,56 @@ def __init__(self, setup_path: str = '/etc/clearpath/') -> None:
             remappings=[('/diagnostics', 'diagnostics'),],
         )
 
+        # Onboard router & base station
+        if isinstance(self.clearpath_config.platform.wireless.router, PeplinkRouter):
+            self.wireless_router_node = LaunchFile.Node(
+                package='peplink_router_driver',
+                executable='peplink_router_node',
+                name='router_node',
+                namespace=f'{self.namespace}/network/router',
```

Comment:
Should we namespace this under `wireless/router` instead to keep naming consistent? Wireless watcher should probably be under the same namespace too

Reply:
I was hesitant to change the wireless watcher namespace since that's already been published in our platform API and we may have users (e.g. outdoornav) already relying on that topic as-is.

As for `network/router` and `network/base_station` vs `wireless/router` and `wireless/base_station` I have no strong opinions either way. I chose `network` since it's broader in case we ever expand to non-wireless networking equipment. But that may be unnecessary future-proofing.

Reply:
That is true, some of our MCUs look at the `platform/wifi_connected` topic to control the HMI wifi LED. I think one of the issues we have seen though is that when the robot is not using onboard wifi, the LED will be off even though it is connected through a peplink. It would be nice to have a single topic indicating wifi connection status regardless of source.